mapie/conformity_scores/conformity_scores.py

```diff
@@ -63,38 +69,48 @@ def get_signed_conformity_scores(
 
         Parameters
         ----------
-        y: NDArray
-            Observed values.
+        X: ArrayLike
+            Observed feature values.
 
-        y_pred: NDArray
-            Predicted values.
+        y: ArrayLike
+            Observed target values.
+
+        y_pred: ArrayLike
+            Predicted target values.
 
         Returns
         -------
         NDArray
-            Unsigned conformity scores.
+            Signed conformity scores.
         """
 
     @abstractmethod
     def get_estimation_distribution(
         self,
+        X: ArrayLike,
         y_pred: ArrayLike,
-        conformity_scores: ArrayLike,
+        values: ArrayLike
     ) -> NDArray:
         """
         Placeholder for ``get_estimation_distribution``.
         Subclasses should implement this method!
 
         Compute samples of the estimation distribution from the predicted
-        values and the conformity scores.
+        targets and ``values``that can be either the conformity scores or
+        the conformity scores aggregated with the predictions.
 
         Parameters
         ----------
-        y_pred: NDArray
-            Predicted values.
+        X: ArrayLike
+            Observed feature values.
 
-        conformity_scores: NDArray
-            Conformity scores.
+        y_pred: ArrayLike
+            Predicted reference values of shape (n_samples, ...).
+            The last dimension is the reference of the prediction.
+
+        values: ArrayLike
+            Either the conformity scores or the conformity scores aggregated
+            with the predictions according to the subclass formula.
```

Comment:
UPD: I suggest keeping the previous term: `conformity_scores` to avoid confusion in understanding the method.

Reply:
Done